pages/osx/hashcat.md

```diff
@@ -0,0 +1,24 @@
+# hashcat
+
+> Password recovery utility
+> More information: <https://hashcat.net/wiki/>.
+
+- Wordlist
+
+`hashcat -a 0 -m 400 example400.hash example.dict`
```

Comment:
```suggestion
- Wordlist:

`hashcat --attack-mode={{0}} --hash-type={{400}} {{example400.hash}} {{example.dict}}`
```

Mandatory fix: [broken description](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood): trailing colon and imperative mood expected.
Mandatory fix: [broken option](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax): long option expected.
Mandatory fix: [broken literal](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax): placeholder expected.
Mandatory fix: [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#range-syntax): range syntax expected.